packages/babel-helper-create-class-features-plugin/src/fields.ts

```diff
@@ -669,15 +669,30 @@ const thisContextVisitor = traverse.visitors.merge([
   environmentVisitor,
 ]);
 
+const innerReferencesVisitor = {
+  ReferencedIdentifier(path, state) {
+    const { name } = path.node;
+    if (path.scope.bindingIdentifierEquals(name, state.innerBinding)) {
```

Comment:
can probably just inline the above line vs using destructing if only using it once?
```suggestion
    if (path.scope.bindingIdentifierEquals(path.node.name, state.innerBinding)) {
```